e2e/tests/notebook/notebook-download.spec.ts

```diff
@@ -14,8 +14,8 @@ describe('Jupyter notebook download test', () => {
   const testDownloadModal = async (modal: NotebookDownloadModal): Promise<void> => {
     const checkDownloadDisabledState = async (wantDisabled: boolean) => {
       expect(await waitForFn(async () => {
-        let downloadBtn = await modal.getDownloadButton();
-        return wantDisabled == await getPropValue<boolean>(downloadBtn, 'disabled');
+        const downloadBtn = await modal.getDownloadButton();
+        return wantDisabled === await getPropValue<boolean>(downloadBtn, 'disabled');
```

Comment:
tslint fix

Reply:
The other change is fine, but the `==` was intentional, I'd like to treat nulls as false. If you want to switch this to `===`, please also make it `!!(await getPropValue<boolean>(downloadBtn, 'disabled'))`; Unless you're 100% confident that your helper method properly handles this case.

Also, if there's lint guidance you're following - is it not also part of our presubmit?

Reply:
no presubmit check task at this time. I manually run `yarn lint:fix`.